lib/utils.js

```diff
@@ -1,17 +1,9 @@
 exports.createPromiseCallback = createPromiseCallback;
+var Promise = require('bluebird');
 
 function createPromiseCallback() {
   var cb;
-
-  if (!global.Promise) {
-    cb = function() {};
-    cb.promise = {};
-    Object.defineProperty(cb.promise, 'then', { get: throwPromiseNotDefined });
-    Object.defineProperty(cb.promise, 'catch', { get: throwPromiseNotDefined });
-    return cb;
-  }
-
-  var promise = new global.Promise(function(resolve, reject) {
+  var promise = new Promise(function(resolve, reject) {
     cb = function(err, data) {
       if (err) return reject(err);
       return resolve(data);
```

Comment:
It's fine to leave this since it's in the original code and not part of your changes anyways.